src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -4414,8 +4414,10 @@ private StatementAst UsingStatementRule(Token usingToken)
             {
                 case TokenKind.EndOfInput:
                 case TokenKind.NewLine:
-                // Example, using module ,FooBar
+                // Example: 'using module ,FooBar'
                 case TokenKind.Comma:
+                // Example: 'using module ;'
+                case TokenKind.Semi:
```

Comment:
I don't think `Semi` needs a comment, it's rather obvious, probably even more obvious if you move the case to be next to `NewLine`.